doc/sphinx/source/algorithms.rst

```diff
@@ -92,7 +92,7 @@ These all assume that an object of :class:`.GraphDataScience` is available as `g
 
     Constructs a random subgraph based on random walks with restarts.
 
-.. py:function:: gds.alpha.graph.sample.cnarw(graph_name: str, from_G: Graph, **config: Any)
+.. py:function:: gds.graph.sample.cnarw(graph_name: str, from_G: Graph, **config: Any)
```

Comment:
Do we not also need to add the new estimate endpoint here? :thinking:

Reply:
We should have tests that catch this

Reply:
Looks like its mentioning was missed in `example_server_endpoints.py` file. Thank you for finding, I will fix